package.json

```diff
@@ -63,6 +63,7 @@
     "test:unit:coverage": "vitest run --coverage",
     "test:unit:update-snapshots": "vitest --update",
```

Comment:
☝️

Reply:
What is your suggestion here? To add the build step here as well?

It's also not exactly shorter, I may as well be typing out the whole thing at that point ;)